apps/11gist-blog/script.js

```diff
@@ -1,5 +1,48 @@
 'use strict';
 
 $(document).ready(function() {
-  // You code here
+    // You code here
+    function formatDate(date) {
+        var arrDate = date.split('-');
+        return arrDate[1] + '/' + arrDate[2] + '/'
+        arrDate[0]
+    }
+
+    $.ajax('https://api.github.com/users/sanderson60/gists', {
+        success: function getGists(response) {
+            response.forEach(function(post) {
+                if (post.description.match('^#post')) {
+                    var postDesc = post.description.replace(/#post /g, '');
```

Comment:
From what I researched the g modifier is used to perform a global match (find all matches rather than stopping after the first match). It appears to work this way, am I wrong?